docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,206 @@ description: guide for doing a lift and shift migration of airflow to dagster
   !
 </Note>
 
-Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow to Dagster without any rewriting.
+Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow without any rewriting.
 
-This guide will walk you through the steps of doing this migration.
+This guide will walk you through the steps of performing this migration.
 
 ---
 
 ## Prerequisites
 
-To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+Before beginning a migration it is important for you to do some upfront analysis:
+
+- **To understand the complexity of undertaking a migration**. Refer to the [previous section](#before-you-begin) for more detail.
+
```

Comment:
This reads strangely now, because the first sentence in this section doesn't relate/lead into every one of the bullet points in this section.

Also changing `previous section` to `next section` since that's where it wound up being placed.

Suggestion:

```suggestion
To complete the migration, you'll need:

- **To perform some upfront analysis**. Refer to the [next section](#before-you-begin) for more detail.

```